Tools/autotest/unittest/extract_param_defaults_unittest.py

```diff
@@ -71,7 +71,7 @@ def test_logfile_does_not_exist(self, mock_mavlink_connection):
 
         # Call the function with a dummy logfile path
         with self.assertRaises(SystemExit) as cm:
-            extract_parameter_default_values('dummy.bin')
+            extract_parameter_values('dummy.bin')
```

Comment:
This is awkward.  You haven't added the keyword argument here, so this doesn't *actually* get the parameter values, does it?  It gets the default values?

Might I suggest adding an entry point to the library instead?

So leave `extract_parameter_default_values` and add `extract_parameter_values`as a separate call.

Reply:
The default is to test default values